tests/adapters/switches/juniper_test.py

```diff
@@ -4297,6 +4297,68 @@ def test_enable_interface_succeeds(self):
 
         self.switch.set_interface_state("ge-0/0/6", ON)
 
+    def test_reset_interface_succeeds_removing_enabled_state(self):
+
+        self.netconf_mock.should_receive("edit_config").once().with_args(target="candidate", config=is_xml("""
+            <config>
+              <configuration>
+                <interfaces>
+                  <interface>
+                    <name>ge-0/0/6</name>
+                    <enable />
```

Comment:
it has come to our ears that the enable here is not a real attribute is it is faulty to use it, therefore this method should only remove a <disable> node and i think you need to be sure it is there, please verify